components/announcement_bar/text_dismissable_bar.tsx

```diff
@@ -34,7 +34,7 @@ export default class TextDismissableBar extends React.PureComponent<Props, State
     }
 
     static getDerivedStateFromProps(props: Props) {
-        const dismissed = localStorage.getItem(localStoragePrefix + props.text);
+        const dismissed = localStorage.getItem(localStoragePrefix + props.text?.toString());
```

Comment:
Are you sure this works? I don't even know what calling `toString` on a React element would do

Reply:
@hmhealey it is an object so the result would be '"[object object]"'

Reply:
I just saw this one as well https://stackoverflow.com/a/71846895/4159720